spec/ulid_spec.rb

```diff
@@ -4,8 +4,11 @@
 #               --------------------------
 #               0DHWZ1DT60
 #                         60ZVCSJFXBTG0J2N
-KNOWN_STRING = '01ARYZ6RR0T8CNRGXPSBZSA1PY'
-KNOWN_TIME = Time.parse('2016-07-30 22:36:16 UTC')
+KNOWN_STRING = "01ARYZ6RR0T8CNRGXPSBZSA1PY"
+KNOWN_TIME = Time.parse("2016-07-30 22:36:16 UTC")
+KNOWN_UUID = "01563df3-6300-d219-5c43-b6caff9506de"
+KNOWN_BYTES = "\x01V=\xF3c\x00\xD2\x19\\C\xB6\xCA\xFF\x95\x06\xDE".b
+KNOWN_UINT128 = 1777022035688232904178850488005232350
```

Comment:
I'm assuming we're fine with this being stored under-the-hood as a variable-length Integer (~ BigNum)?

<img width="453" alt="image" src="https://github.com/Shopify/ulid-ruby/assets/9891776/e040262a-feb6-49b4-be26-30e83f9ab1bc">